webview/src/setup/components/studio/Connect.tsx

```diff
@@ -30,7 +30,8 @@ export const Connect: React.FC = () => {
           />
         </div>
         <p>
-          Don&apos;t Have an account? <a href={STUDIO_URL}>Get Started</a>
+          Don&apos;t Have an account?{' '}
+          <a href={DEFAULT_STUDIO_URL}>Get Started</a>
```

Comment:
Currently, next steps were adding a Command Pallete command for updating the URL and maybe adding text about it in the studio section somewhere. Not sure where would be good spot yet 🤔